### DISCUSSION TITLE
Welcome to externpro Discussions!

### DISCUSSION THREAD

**Discussion by smanders** [smanders/externpro#290] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by briday1** [comment]


**Reply by smanders** [reply]
I'm interested to know what code you've inherited that uses both externpro and internpro. In a nutshell, externpro is public and builds external projects, while internpro is private to Space Dynamics Lab and builds internal projects.
